apps/nextjs-website/src/editorialComponents/Feature/FeatureStackItem.tsx

```diff
@@ -49,25 +54,37 @@ export const FeatureStackItem = ({
         }}
         color={imageStyle}
       >
-        <IconWrapper size={64} color={imageStyle} icon={item.iconName} />
+        {item.iconName && (
+          <IconWrapper size={64} color={imageStyle} icon={item.iconName} />
+        )}
+        {item.iconUrl && (
+          <Image src={item.iconUrl} alt={item.title} width={64} height={64} />
+        )}
       </Box>
       <Stack color={textStyle} spacing={1} textAlign='center'>
         <Typography color='inherit' variant='h6'>
           {item.title}
         </Typography>
         <>
-          {!item.link ? (
-            <Typography variant='body2' color='inherit'>
-              {item.subtitle}
-            </Typography>
-          ) : (
-            <Subtitle
-              isDarkMode={isDarkMode}
-              subtitle={item.subtitle}
-              textLink={item.link.text}
-              url={item.link.url}
+          {item.subtitle ? (
+            !item.link ? (
+              <Typography variant='body2' color='inherit'>
+                {item.subtitle}
+              </Typography>
+            ) : (
+              <Subtitle
+                isDarkMode={isDarkMode}
+                subtitle={item.subtitle}
+                textLink={item.link.text}
+                url={item.link.url}
+              />
+            )
+          ) : item.content ? (
+            <BlocksRendererClient
+              content={item.content}
+              paragraphSx={{ color: 'white' }}
             />
-          )}
+          ) : null}
```

Comment:
I thing the double ternary operator makes this a little bit difficult to read

you could replace the first with a `&&` given that the : option is a null

```
          {item.subtitle && (
            !item.link ? (
              <Typography variant='body2' color='inherit'>
                {item.subtitle}
              </Typography>
            ) : (
              <Subtitle
                isDarkMode={isDarkMode}
                subtitle={item.subtitle}
                textLink={item.link.text}
                url={item.link.url}
              />
            )
          ) : item.content ? (
            <BlocksRendererClient
              content={item.content}
              paragraphSx={{ color: 'white' }}
            />
          ) }
```

Or, maybe put it in a separate component